vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -3,53 +3,84 @@
     set_fact:
       NUMBER_OF_BMH: "{{ NUM_OF_MASTER_REPLICAS|int +  NUM_OF_WORKER_REPLICAS|int }}"
 
-  - name: Create directories for storing container logs
-    file:
-      path: "/tmp/{{ CONTAINER_RUNTIME }}/{{ item }}"
-      state: directory
-    loop:
-        - ironic-api
-        - ironic-conductor
-        - ironic-inspector
-        - dnsmasq
-        - httpd-infra
-        - mariadb
-        - ironic-endpoint-keepalived
-        - ironic-log-watch
-    when: EPHEMERAL_CLUSTER == "kind"
+  - name: Fetch container logs (kind cluster)
+    block:
+
+      - name: Create directories for storing container logs (kind cluster)
+        file:
+          path: "/tmp/{{ CONTAINER_RUNTIME }}/{{ item }}"
+          state: directory
+        loop:
+            - ironic-api
+            - ironic-conductor
+            - ironic-inspector
+            - ironic-endpoint-keepalived
+            - ironic-log-watch
+            - dnsmasq
+            - httpd-infra
+            - mariadb
+            - registry
+            - sushy-tools
+            - vbmc
+
+      - name: Fetch container logs before pivoting (kind cluster)
+        shell: "sudo {{ CONTAINER_RUNTIME }} logs {{ item }} > /tmp/{{ CONTAINER_RUNTIME }}/{{ item }}/stdout.log 2> /tmp/{{ CONTAINER_RUNTIME }}/{{ item }}/stderr.log"
```

Comment:
Is `2>` in this case is getting the error encountered during log collection process ? or trying to separate logs into stdout and stderr ?

Reply:
the latter option